aopy/analysis/base.py

```diff
@@ -1093,44 +1106,63 @@ def calc_mt_tfr(ts_data, n, p, k, fs, step=None, fk=None, pad=2, ref=True, compl
         ts_data = np.array(ts_data)
     if ts_data.ndim == 1:
         ts_data = ts_data[:, np.newaxis]
+    if ts_data.ndim == 2:
+        ts_data = ts_data[:, :, np.newaxis]
     if ts_data.shape[1] == 1:
```

Comment:
Input of [nt, nch] array is still allowed, right? Specifying this in docstrings is helpful because someone wants to still use [nt, nch] array without using trial segmented data, considering computation speed.